### ISSUE TITLE
Dependabot repeatedly raises pull requests for the same dependency

### ISSUE THREAD

**Issue by matthewfranglen** [dependabot/feedback#808]
Dependabot opens a PR to upgrade python Pillow to 6.2.0.
I take the time to perform that upgrade, with associated dependencies, and then test it. I merge the fix.
Dependabot _immediately_ opens another PR to upgrade python Pillow to 6.2.1.
I hate it.

**Comment by feelepxyz** [comment]
@matthewfranglen this is currently expected behaviour because versions below `6.2.0` are vulnerable meaning the first update is attempted as the lowest possible update to a non-vulnerable version. We've done this to make security updates easier to merge with less potential breaking changes. Once this is merged Dependabot tries to update to the latest version if you have enabled all updates.

Details of the vulnerability fixed in `6.2.0`: https://github.com/advisories/GHSA-j7mj-748x-7p78

**Comment by matthewfranglen** [comment]
6.2.0 is also vulnerable though. I have finite time to check this stuff so if you are going to raise a PR to fix a dependency then please upgrade it to something that does not then trigger yet another dependabot PR.

**Comment by feelepxyz** [comment]
> 6.2.0 is also vulnerable though

Oh, do you have a source for this?

**Comment by matthewfranglen** [comment]
https://github.com/matthewfranglen/twitter-word-to-vec/pull/6

This is a dependabot PR for the Pillow dependency which upgrades it from 6.2.0 to 6.2.1.
It was raised 26 Dec 2019 20:39 GMT.
I had merged https://github.com/matthewfranglen/twitter-word-to-vec/pull/5 which was my upgrade to the Pillow dependency to version 6.2.0. I merged that on 26 Dec 2019 20:38 GMT.

I can't confirm if Pillow 6.2.0 is vulnerable. My assumption is that dependabot is raising PRs for security issues, so the PR is the source. Version 6.2.1 of Pillow was released on 21 Oct 2019 which predates the original pull request by more than a month. So I got punished for resolving that PR - it cost me time and it did not stop dependabot from raising another PR.

**Comment by feelepxyz** [comment]
@matthewfranglen sorry for the slow response. I've gotten to the bottom of this, it looks like this was caused by a race-condition where the correct PR updating to 6.2.0 was rebased at the same time as you closed the pull request causing the rebase to open a new pull request instead of dismissing it. Will get this fixed.
